.github/workflows/add-issues-to-project.yml

```diff
@@ -0,0 +1,20 @@
+name: Auto Assign to Logs and Traces Project
+
+on:
+  issues:
+    types: [opened]
```

Comment:
Should this be just issues or also?
```
pull_request:
    types: [opened]
```

Reply:
And if just issues, then probably this shouldn't have pull requests `Assign NEW issues and NEW pull requests to Logs&Traces project`

Reply:
Good idea! I added it to the file!